spec/attime_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'attime'
+
+describe '#attime' do
+  subject do |example|
+    delta = example.metadata[:description_args][0]
+    CheckGraphite.attime(delta)
+  end
+
+  def timedelta(n)
+    Time.now + n
+  end
+
+  it("10days") { should be_within(0.9).of(timedelta(10 * 86400)) }
+  it("0days") { should be_within(0.9).of(timedelta(0)) }
+  it("-10days") { should be_within(0.9).of(timedelta(-10 * 86400)) }
+  it("5seconds") { should be_within(0.9).of(timedelta(5)) }
+  it("5minutes") { should be_within(0.9).of(timedelta(5 * 60)) }
+  it("5hours") { should be_within(0.9).of(timedelta(5 * 3600)) }
+  it("5weeks") { should be_within(0.9).of(timedelta(86400 * 7 * 5)) }
+  it("1month") { should be_within(0.9).of(timedelta(30 * 86400)) }
+  it("2months") { should be_within(0.9).of(timedelta(60 * 86400)) }
+  it("12months") { should be_within(0.9).of(timedelta(360 * 86400)) }
+  it("1year") { should be_within(0.9).of(timedelta(365 * 86400)) }
+  it("2years") { should be_within(0.9).of(timedelta(730 * 86400)) }
```

Comment:
You would actually get `"10days" => be_within(0.9).of(10 * 86400)`. The proposed style breaks the rules that tests should be reasonably independent and I do not think it really improves readability.